lint/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+# This file is part of sv-witnesses repository: https://github.com/sosy-lab/sv-witnesses
+#
+# SPDX-FileCopyrightText: 2021 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[tool.poetry]
+name = "witnesslint"
+version = "1.3"
```

Comment:
I would suggest to not mix bumping the version with different changes. This will inevitably lead to problems, for example if after merging this we have another change that should go into the release, e.g., because something is broken.

Bumping the version should be done as part of a specific and documented release process.

Reply:
I outlined the release process in `doc/Developing.md`. It is based on the one from benchexec and I made sure that publishing with poetry works as expected with a small example project.